packages/backend/src/queryBuilder.ts

```diff
@@ -52,16 +54,55 @@ const getOperatorSql = (filterGroup: FilterGroup | undefined) => {
     return ' AND ';
 };
 
+export const replaceUserAttributes = (
+    sqlFilter: string,
+    userAttributes: UserAttribute[],
+    stringQuoteChar: string = "'",
+): string => {
+    const userAttributeRegex =
+        /\$\{(?:lightdash|ld)\.(?:attribute|attr)\.(\w+)\}/g;
+    const sqlAttributes = sqlFilter.match(userAttributeRegex);
```

Comment:
We're checking for known attributes but the user might reference an attribute that doesn't exist. 

We could use `liquidjs` like we do in the CLI to parse this reference without having to regex ourselves

Reply:
Sorry my bad!!! I used `attributes` instead of `attribute`

Reply:
now we support also `attributeS`